crates/typst-cli/src/compile.rs

```diff
@@ -170,9 +170,16 @@ fn export_pdf(document: &Document, command: &CompileCommand) -> StrResult<()> {
     Ok(())
 }
 
+/// Get time passed as environment variable
+fn passed_time() -> Option<NaiveDateTime> {
+    let passed_string = option_env!("SOURCE_DATE_EPOCH")?;
```

Comment:
`option_env` returns variable in compile time, so you should use https://doc.rust-lang.org/stable/std/env/fn.var.html